options/locale/locale_en-US.ini

```diff
@@ -1559,6 +1559,14 @@ pulls.squash_merge_pull_request = Create squash commit
 pulls.merge_manually = Manually merged
 pulls.merge_commit_id = The merge commit ID
 pulls.require_signed_wont_sign = The branch requires signed commits but this merge will not be signed
+pulls.merge_pull_request_now = Merge Pull Request Now
+pulls.rebase_merge_pull_request_now = Rebase and Merge Now
+pulls.rebase_merge_commit_pull_request_now = Rebase and Merge Now (--no-ff)
+pulls.squash_merge_pull_request_now = Squash and Merge Now
+pulls.merge_pull_request_on_status_success = Merge Pull Request When All Checks Succeed
+pulls.rebase_merge_pull_request_on_status_success = Rebase and Merge When All Checks Succeed
+pulls.rebase_merge_commit_pull_request_on_status_success = Rebase and Merge (--no-ff) When All Checks Succeed
+pulls.squash_merge_pull_request_on_status_success = Squash and Merge When All Checks Succeed
```

Comment:
Many uppercase letters, are they by purpose? Maybe something like `Rebase and Merge When All Checks Succeed` => `Rebase and Merge when all checks succeed`?

(If they are by purpose, that's fine)